docs/src/index.tmpl

```diff
@@ -88,17 +88,23 @@ function setup_page(){
   <img src="linuxcnc-logo-chips.png" alt="LinuxCNC Logo" width="175"/>
 </div>
 
-<h3>LinuxCNC version <strong>@VERSION@</strong></h3>
+<!-- <h3 align="center">LinuxCNC version <strong>@VERSION@</strong></h3>
+<h2>Documentation</h2> -->
```

Comment:
Why is this part commented out?

Reply:
Good catch, these lines should actually be removed.